core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -554,6 +555,32 @@ public String authority() {
     }
   }
 
+  private final class DirectFallbackExecutor implements Executor {
```

Comment:
Seems previously we also would use the `executor` after shutdown: anything inside ClientCallImpl.start() may be happening after shutdown.